bin/run-tests

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+for D in exercises/*; do
+    CURRENT_DIR=$(pwd)
+
+    if [ -d "${D}" ]; then
+        # Get the exercise name from the test file
+        TEST_FILE=$(basename $(ls ${D}/test/test*))
+        STRIPPED_OF_EXTENSION="${TEST_FILE%.*}"
+        EXERCISE_NAME="${STRIPPED_OF_EXTENSION:5}"
+
+        # Copy the examples with the correct name for the exercise
+        if [ -e "${D}/src/example.c" ]
+        then
+          cp -n ${D}/src/example.c ${D}/src/${EXERCISE_NAME}.c
```

Comment:
Remove the `-n` from both `cp` commands. We want them to override the files, like with our dummpy `hello_world.c`